src/refer.ts

```diff
@@ -81,32 +101,26 @@ function refer(this: any, options: any) {
     }
   }
 
-
   async function prepare(this: any) {
-    let seneca = this
+    const seneca = this
     await seneca.post('biz:refer,load:rules')
```

Comment:
there is a bug I just found in seneca-msg-test that will prevent this from working - make sure to use the latest version

Reply:
Understood, lib version updated.